specification/codesigning/resource-manager/readme.python.md

```diff
@@ -0,0 +1,24 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+```yaml $(python)
+python-mode: create
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: azure.mgmt.codesigning
+package-name: azure-mgmt-codesigning
+package-version: 0.1.0
+clear-output-folder: true
+```
+
+``` yaml $(python) && $(python-mode) == 'update'
+no-namespace-folders: true
+output-folder: $(python-sdks-folder)/codesigning/azure-mgmt-codesigning/azure/mgmt/codesigning
+```
+
+``` yaml $(python) && $(python-mode) == 'create'
+basic-setup-py: true
+output-folder: $(python-sdks-folder)/codesigning/azure-mgmt-codesigning
+```
```

Comment:
please follow latest template https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/samplefiles/readme.python.md to update this file

Reply:
Updated